Tasks/GitHubReleaseV0/README.md

```diff
@@ -34,11 +34,11 @@ The GitHub Releases task can run in 3 action types, viz. create, edit or delete.
 
 * **Target:** This is the commit SHA for which the GitHub release will be created. By default, the value is $(Build.SourceVersion) which corresponds to the commit for which the build was run. If you specify a branch name here(E.g. *master* ), the latest commit from this branch will be used as target. This field is ignored when using edit and delete actions.
 
-* **Tag Source:** This field allows you to configure the tag to be used for a release action. It can be done in 2 ways:
+* **Tag Source:** This field allows you to choose the tag to be used for a release action. It can be done in 2 ways:
 
-    * Git tag: This option is available only for *create* action. If selected, the release will be created using the git tag that is associated with this commit. If no tag is found for the given commit, the release will not be created. If multiple tags are found, the task will throw an error.
+    * Git tag: Choose this option if you want use the tag that is pushed along with the commit. The release will be created using the git tag that is associated with this commit. If no tag is found for the given commit, the release will not be created. If multiple tags are found, the task will throw an error.This option is available only for *create* action.
 
-    * User specified tag: When this option is used, the release will be created using the tag mentioned. You can also mention the tag using variables, Eg. v\$(MajorVersion).\$(MinorVersion).\$(PatchVersion). For release edit and delete actions, it is mandatory to use this option.
+    * User specified tag: Choose this option if you want the task to create a new tag. The release will subsequently be created using this new tag. You can specify the tag to be created directly or using variables, Eg. v\$(MajorVersion).\$(MinorVersion).\$(PatchVersion). If the tag already exists in the repo, then the release will be created for the existing tag. This option is available for *create*, *edit* or *delete* actions. However, it is the only tag source option for *edit* and *delete* actions.
```

Comment:
For edit and delete actions Tag Source option is not even available. You need to modify this